src/applications/health-care-supply-reordering-redirect/tests/e2e/redirect.cypress.spec.js

```diff
@@ -0,0 +1,31 @@
+import { appName, rootUrl } from '../../manifest.json';
+
+describe(`${appName} -- Redirect`, () => {
+  // eslint-disable-next-line @department-of-veterans-affairs/axe-check-required
+  // Need to skip the test until the content-build registry is updated to run the app.
```

Comment:
Note: Need to skip the tests because the content-build registry has not been updated, but need the app merged in for the registry to be updated.